tendermint/src/error.rs

```diff
@@ -40,6 +40,58 @@ pub enum Kind {
     /// Signature invalid
     #[error("bad signature")]
     SignatureInvalid,
+
+    /// invalid Vote type
+    #[error("invalid Type")]
+    InvalidMessageType,
+
+    /// Negative block height
+    #[error("negative height")]
+    NegativeHeight,
+
+    /// Negative voting round
+    #[error("negative round")]
+    NegativeRound,
+
+    /// Negative POL round
+    #[error("negative pol round")]
+    NegativePOLRound,
+
+    /// Negative validator index in vote
+    #[error("negative ValidatorIndex")]
+    NegativeValidatorIndex,
+
+    /// Wrong hash size in part_set_header
+    #[error("Wrong hash: expected Hash size to be 32 bytes")]
+    InvalidHashSize,
+
+    /// No timestamp in vote
+    #[error("no timestamp")]
+    NoTimestamp,
+
+    /// Invalid account ID length
+    #[error("invalid account ID length")]
+    InvalidAccountIDLength,
```

Comment:
```suggestion
    InvalidAccountIdLength,
```

Convention in Rust is to use strict CamelCase (I know, it looks weird…)

Reply:
Covered in the next commit.